### DISCUSSION TITLE
please update the fltapak version or add it to the ubuntu repo

### DISCUSSION THREAD

**Discussion by theArianit** [lapce/lapce#2829] [General]
Please update lapce flatpak or add it to ubuntu's repo, since ubuntu and its derivates, have the biggest market share of linux distros.

**Comment by panekj** [comment]
Lapce Flatpak is not managed by Lapce maintainers (except me) so it's more appropriate to open issues in flathub repository

**Reply by theArianit** [reply]
Thanks for replying. I will keep that in mind for next time.
